pom.xml

```diff
@@ -48,6 +48,7 @@
         <kafka.connect.maven.plugin.version>0.11.1</kafka.connect.maven.plugin.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <commons.codec.version>1.15</commons.codec.version>
+        <snakeyaml.version>1.28</snakeyaml.version>
```

Comment:
description says `1.29` is this a typo?

Reply:
Description is a typo. `1.28` is the latest version